csharp/src/Apache.Arrow.Flight/Apache.Arrow.Flight.csproj

```
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Google.Protobuf" Version="3.14.0" />
+    <PackageReference Include="Grpc.Net.Client" Version="2.33.1" />
+    <PackageReference Include="Grpc.Tools" Version="2.33.1" PrivateAssets="All" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Apache.Arrow\Apache.Arrow.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Protobuf Include="..\..\..\format\Flight.proto" />
```

```suggestion
    <Protobuf Include="..\..\..\format\Flight.proto" Access="internal" />
```

Let's generate the "Protocol" classes as `internal`, as we don't want public callers to use these types directly.

---

All the protocol classes are now internal.